src/content/docs/more-integrations/open-source-telemetry-integrations/opentelemetry/get-started/opentelemetry-tutorial-python.mdx

```diff
@@ -40,49 +40,64 @@ import opentelemetryPythonLogCapturedinTraceSpan from 'images/opentelemetry_scre
 
 
 
-<SideBySide>
-  <Side>
-    Try out any of these Python tutorials to see what the New Relic platform can do with your OTLP data. We have three tutorials you can choose from, each one using the same demo Flask app. The app will calculate the nth number in the Fibonacci sequence and generate traces, metrics, and logs.
+Try out any of these Python tutorials to see what the New Relic platform can do with your OTLP data. We have three tutorials you can choose from, each one using the same demo Flask app. The app will calculate the nth number in the Fibonacci sequence and generate traces, metrics, and logs.
```

Comment:
Should the 'nth' be in italics or something to differentiate it? Also as vale says, we have > we've

Reply:
@jeff-colucci I don't see any special formatting of nth in math tutorials. Let's keep this. I am moving all these changes to a new PR since this one became out of date because of some SME changes.